$NetBSD: patch-aa,v 1.1.1.1 2009/01/10 21:08:14 jmcneill Exp $

NetBSD doesn't provide v4l1 compatibility, so use libv4l1 until
frameworks is ported to v4l2.

--- configure.in.orig	2007-02-27 22:18:10.000000000 -0500
+++ configure.in
@@ -12,7 +12,7 @@ AC_PROG_CC
 AC_PROG_INSTALL
 
 # Checks for libraries
-pkg_modules="gtk+-2.0 gthread-2.0 libglade-2.0"
+pkg_modules="gtk+-2.0 gthread-2.0 libglade-2.0 libv4l1"
 PKG_CHECK_MODULES(PACKAGE, [$pkg_modules])
 AC_SUBST(PACKAGE_CFLAGS)
 AC_SUBST(PACKAGE_LIBS)
